src/uv_fs.c

```diff
@@ -529,7 +529,7 @@ static int probeDirectIO(int fd, size_t *size, char *errmsg)
         switch (fs_info.f_type) {
             case 0x01021994: /* TMPFS_MAGIC */
             case 0x2fc12fc1: /* ZFS magic */
-            case 0x24051905: /* UBIFS Suprt magic */
+            case 0x24051905: /* UBIFS Support magic */
```

Comment:
It's possible this is wrong, but ...
I also wonder if the word is necessary at all.

Reply:
not strictly needed indeed but that's fine